catalog/ui/catalog-ui-search/package.json

```diff
@@ -64,6 +65,7 @@
     "squirejs": "0.2.1",
     "stack-source-map": "1.0.5",
     "webpack": "1.13.0",
+    "webpack-bundle-size-analyzer": "^2.5.0",
```

Comment:
We need to remove the version range modifier `^` if we need to keep the `webpack-bundle-size-analyzer` in the project.